src/photos/ducks/backup/index.jsx

```diff
@@ -33,7 +33,7 @@ const BackupPage = () => {
             <BackupInfo />
             {!isFlagshipApp() ? <InstallAppAlert /> : null}
             <BackupActions />
-            <BackupDescription />
+            {isMobile ? <BackupDescription /> : null}
```

Comment:
nit: You can also use `&&` to be less verbose
```
{isMobile && <BackupDescription />}
```

Reply:
Agree about less verbose but trying this way for the next weeks https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx (still not sure about the value but trying 😄)

Reply:
Thanks for the article 🙂 I use this way also now to avoid having `0` or `NaN` popping randomly